Modules/Filtering/ImageIntensity/test/itkSymmetricEigenAnalysisImageFilterTest.cxx

```diff
@@ -120,6 +121,8 @@ class SymmetricEigenAnalysisImageFilterHelper : public SymmetricEigenAnalysisIma
     filter->SetFunctor(filter->GetFunctor());
 
     filter->OrderEigenValuesBy(order);
+    filter->SetOrderEigenValuesBy(order);
+    // ITK_TEST_SET_GET_VALUE(order, filter->GetDimension());
```

Comment:
Sorry to comment on an already merged PR, could not have a look before: is testing `GetOrderEigenValuesBy` here being added in a separate PR?